threescale_api/resources.py

```diff
@@ -490,6 +490,7 @@ def url(self) -> str:
     def update(self, params: dict = None, **kwargs) -> 'DefaultResource':
         return self.rest.patch(url=self.url, json=params, **kwargs)
 
+
```

Comment:
this renaming doesn't seem to be relevant to the change in this PR, rather shouldn't be in this PR

Reply:
Fixed